util/printer/printer_test.go

```diff
@@ -16,27 +16,16 @@ package printer
 import (
 	"testing"
 
-	. "github.com/pingcap/check"
-	"github.com/pingcap/tidb/util/testleak"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	TestingT(t)
-}
-
-var _ = Suite(&testPrinterSuite{})
-
-type testPrinterSuite struct {
-}
-
-func (s *testPrinterSuite) TestPrintResult(c *C) {
-	defer testleak.AfterTest(c)()
+func TestPrintResult(t *testing.T) {
+	t.Parallel()
 	cols := []string{"col1", "col2", "col3"}
 	datas := [][]string{{"11"}, {"21", "22", "23"}}
 	result, ok := GetPrintResult(cols, datas)
-	c.Assert(ok, IsFalse)
-	c.Assert(result, Equals, "")
+	assert.False(t, ok)
```

Comment:
`assert.False` -> `require.False`?

Reply:
We need using `require` instead of `assert`. @jyz0309

Reply:
tisonkun had used `assert` in the early migration PRs for most assertions but important assertions. And I am not so against with it.

Reply:
Done. PTAL~

Reply:
@mmyj @xhebox now I tend to use `require` which keeps the same semantic with `c.Assert`. Early migration is lack of best practice and I think I didn't get it better then. It is not a tech blocker if there is no effective difference between calling `Fail` and calling `FailNow`, but I tend to fail test fast.